Update README instructions (#143)
AlexsLemonade/training-specific-template: README.md

# training-specific-template

This repository is intended to serve as a template for creating a repository for an individual Data Lab workshop.
The repository structure and use of GitHub Pages is intended to gather all material required to administer a workshop in one user-friendly place.
We use a template repository approach for maintainability.

## Creating a repository for an individual training workshop

First, you will need to create a new repository using the [`Use the template` button](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template).
This will create a new repository with the same layout as this template repository under the user or organization you choose as the owner of the repository; if you are a Data Lab member running an internal training workshop, the `AlexsLemonade` organization should be the owner.

If you are using GitHub from a paid tier account (e.g., GitHub Pro, GitHub Team, or GitHub Enterprise), you can set the visibility of this repository as private; the associated GitHub Pages website will still be public.

> However, if you are using GitHub from a _free_ account (e.g., GitHub Free for individuals or organizations), you must set the visibility of this repository as public in order for GitHub Pages to be visible, as described in this [documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages).

Reviewer: Slight wording change to remove double visible/visibility. And a slightly bigger link target.
```suggestion
However, if you are using GitHub from a _free_ account (e.g., GitHub Free for individuals or organizations), you must set the visibility of this repository as public in order for GitHub Pages to be active, as described in [this documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages).
```

## Creating a repository for an individual training workshop

First, you will need to create a new repository using the [`Use the template` button](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template).

> This will create a new repository with the same layout as this template repository under the given organization or user you choose to administer the repository; if you are a Data Lab member running an internal training workshop, the `AlexsLemonade` organization should administer the new repository.

Reviewer: You will be set as an administrator (and can add others), but the org is the Owner, which is what the menu you select from shows.
```suggestion
This will create a new repository with the same layout as this template repository under the user or organization you choose as the owner of the repository; if you are a Data Lab member running an internal training workshop, the `AlexsLemonade` organization should be the owner.
```

We recommend setting the new repository to public.

> (When you use GitHub pages on a private repository, those pages are publicly accessible anyway.)

Reviewer: So, this is tricky: for outside users without paid plans, the repo _must_ be public to use Pages. I don't know if we want to get into it more than we did before. (But also, capitalize "Pages"?)

Author: First I have a typo there - should be "setting the new repo to **private**."

That said, I don't think what you wrote it true?

https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages

> Warning: GitHub Pages sites are publicly available on the internet, even if the repository for the site is private. If you have sensitive data in your site's repository, you may want to remove the data before publishing. For more information, see "[About repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories#about-repository-visibility)."

Reviewer: Yes but also (from the green box): 
> GitHub Pages is available in public repositories with GitHub Free and GitHub Free for organizations, and in public and private repositories with GitHub Pro, GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server.

Author: Ah, I see... yeah..

Author: I've added more detail here, let me know what you think!

## Customizing the new repository for an individual training workshop

We have set up a [manually-triggered GitHub Action](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) to support setting up this new repository.
When triggered, this action will file a series of issues comprising tasks that should be completed before the start of a workshop.
You should trigger this action once the [`training-modules` repository](https://github.com/AlexsLemonade/training-modules) tag for the given workshop has been created, as one of the _first steps_ of setting up your new repository.

To run this action, navigate to the "Actions" tab.
On the left, you will see all available workflows.
Click the workflow named `Manually trigger issue creation for standard set up`, and then click the "Run workflow" dropdown button.

> You will need to provide one required input, the `training-modules` repository release tag you would like to associate with the workshop.

Reviewer: Can this be updated later? If so, we should say that here, and probably mention the default. Or if not, we might say whether this release tag needs to exist, or if it is the one we _will_ assign?

Author: Hm, looks like the _only_ time this tag is used is for the title of an issue!

https://github.com/AlexsLemonade/training-specific-template/blob/615e672511c8abf0e2ef92d62f98649851c5e2dd/.github/workflows/file-setup-issues.yml#L80-L86

I think it's worth just getting rid of this input since there's enough info in the issue template itself to create the release:
https://github.com/AlexsLemonade/training-specific-template/blob/main/setup-issue-templates/workshop-release.md
